notebooker/_entrypoints.py

```diff
@@ -64,6 +64,12 @@ def filesystem_default_value(dirname):
     is_flag=True,
     help="If selected, notebooker will not try to pull the latest version of python templates from git.",
 )
+@click.option(
+    "--execute-at-origin",
+    default=False,
+    is_flag=True,
+    help="If selected, notebooker will execute the notebook at the origin to keep original context",
```

Comment:
I think this helpstring could be a little more descriptive - what is the "origin" from the perspective of the caller?

Reply:
Change, please check